modules/top-level/files/default.nix

```diff
@@ -15,6 +15,16 @@ let
     modules = lib.optionals (!config.isDocs) [
       ../../.
       ./submodule.nix
+      # Pass module args through to the submodule (except `name`)
+      # Wrap each arg with the correct priority
+      {
+        _file = ./.;
```

Comment:
I think `types.submodule` will apply a "default file" gleaned from the submodule options's declaration location.

Testing this out by removing `_file` and adding an error (`foo = "bar"`):

```
error: The option `files."after/ftplugin/python.lua".foo' does not exist. Definition values:
- In `/nix/store/XXX-source/modules/top-level/files': "bar"
Test failure
```

```suggestion
```